velox/functions/prestosql/registration/JsonFunctionsRegistration.cpp

```diff
@@ -26,15 +26,15 @@ void registerJsonFunctions(const std::string& prefix) {
       {prefix + "is_json_scalar"});
   registerFunction<JsonExtractScalarFunction, Varchar, Json, Varchar>(
       {prefix + "json_extract_scalar"});
-  registerFunction<JsonArrayLengthFunction, int64_t, Json>(
+  registerFunction<SIMDJsonArrayLengthFunction, int64_t, Json>(
```

Comment:
Should we now remove JsonArrayLengthFunction and other no longer used functions?